src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2049,6 +2050,18 @@ namespace Akka.Actor.Internal
         public UnboundedStashImpl(Akka.Actor.IActorContext context) { }
     }
 }
+namespace Akka.Actor.Scheduler
+{
+    public interface ITimerScheduler
+    {
+        void Cancel(string key);
+        void CancelAll();
+        bool IsTimerActive(string key);
```

Comment:
Original impleentation uses `object` keys. Any particular reason to use strings?

Reply:
changed to `object`